CONTRIBUTING.adoc

```diff
@@ -0,0 +1,90 @@
+= Contributing to Kubernetes the AWSome Way
+Thank you for your interest in contributing to Kubernetes the AWSome Way! We work hard to provide a high quality and useful workshop for our customers, and we appreciate your interest in helping us and the rest of our community of users. We welcome bug reports, enhancements, and contributions.
+
+__Jump To:__
+
+* link:#bug-reports[Bug Reports]
+* link:#enhancements[Enhancements]
+* link:#contributions[Contributions]
+* link:#getting-in-contact[Getting in Contact]
+
+== Bug Reports
+Bug reports are accepted through the link:https://github.com/aws-samples/aws-workshop-for-kubernetes/issues[Issues] page.
+
+The following labels are used to track bug related issues: link:https://github.com/aws-samples/aws-workshop-for-kubernetes/labels/Bug[Bug].
```

Comment:
Are we going to retrofit all the existing issues with this label?

Reply:
The manual way.  Working on that over the next few days.